pkg/operator/controller/crl/controller.go

```diff
@@ -38,6 +38,8 @@ const (
 
 var log = logf.Logger.WithName(controllerName)
 
+var currentCRLUpdate time.Time
```

Comment:
Is using a single global variable going to work properly when there are multiple ingresscontrollers with CRLs?